tests/Doctrine/Tests/ORM/Functional/Ticket/DDC3192Test.php

```diff
@@ -155,7 +155,7 @@ public function convertToDatabaseValue($value, AbstractPlatform $platform)
      */
     public function convertToPHPValue($value, AbstractPlatform $platform)
     {
-        return array_search($value, self::$map);
+        return array_search($value, self::$map, true);
```

Comment:
This change has actually broken the test:
```
1) Doctrine\Tests\ORM\Functional\Ticket\DDC3192Test::testIssue
Failed asserting that false matches expected 'BYR'.

/www/doctrine/doctrine2/tests/Doctrine/Tests/ORM/Functional/Ticket/DDC3192Test.php:55
```

There is likely string to integer coercion that doesn't work after enabling strict mode.

Reply:
Have you reverted the change or should I? Sorry about that!

Reply:
Fixed it in d791f7f6ca77ab9acb282317664d1ea7316bae8d.